### PR TITLE
Admin panel 1 out of 2

### PR DESCRIPTION
**Pull request by margawron**
Implementation for admin panel - OWNER functionality only.
TODO: Make employee functionality.

### CODE REVIEW

File: PhotoPlusBackend/photoplus/.idea/misc.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="11" project-jdk-type="JavaSDK">
```
Comment (syphzur) [inline PhotoPlusBackend/photoplus/.idea/misc.xml]: w h y

File: PhotoPlusBackend/photoplus/src/main/resources/application.yml [hunk 2]
```diff
@@ -33,8 +33,8 @@ spring:
 
   datasource:
     url: jdbc:mysql://localhost:3306/photoPlusDB?useSSL=false&serverTimezone=Europe/Warsaw
-    username: root
-    password: pass
+    username: photoPlus
```
Comment (syphzur) [inline PhotoPlusBackend/photoplus/src/main/resources/application.yml]: omege man why u do this

File: PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/security/custom/CustomBasicAuthenticationFilter.java [hunk 3]
```diff
@@ -44,7 +44,7 @@ public CustomBasicAuthenticationFilter(final AuthenticationManager authenticatio
     protected void doFilterInternal(final HttpServletRequest request, final HttpServletResponse response,
                                     final FilterChain chain) throws IOException, ServletException
     {
-        final String header = request.getHeader(modelPropertiesService.securityHeaderName);
+            final String header = request.getHeader(modelPropertiesService.securityHeaderName);
```
Comment (syphzur) [inline PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/security/custom/CustomBasicAuthenticationFilter.java]: xD

Comment (margawron) [reply PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/security/custom/CustomBasicAuthenticationFilter.java]: Probably pressed tab while debugging. Fixed in 244b7fb

File: PhotoPlusBackend/photoplus/src/main/resources/application.yml (hunk 2 above)
Comment (margawron) [reply PhotoPlusBackend/photoplus/src/main/resources/application.yml]: "I don't want to but i have to." Fixed in ea48623

File: PhotoPlusBackend/photoplus/.idea/misc.xml (hunk 1 above)
Comment (margawron) [reply PhotoPlusBackend/photoplus/.idea/misc.xml]: Just gitignore .idea folder already. Fixed in ea48623

File: PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/controllers/api/UserController.java [hunk 4]
```diff
@@ -34,6 +34,7 @@ public void addLinks(final UserModelDto dto)
         );
     }
 
+
```
Comment (syphzur) [inline PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/controllers/api/UserController.java]: :/

Comment (margawron) [reply PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/controllers/api/UserController.java]: .... a02126b

**Review by syphzur — APPROVED** [review]

**Pull request merged**